src/screens/HyperSwitch/Analytics/PaymentsAnalytics/PaymentAnalytics.res

```diff
@@ -15,7 +15,16 @@ let make = () => {
     try {
       let infoUrl = getURL(~entityName=ANALYTICS_PAYMENTS, ~methodType=Get, ~id=Some(domain), ())
       let infoDetails = await fetchDetails(infoUrl)
-      setMetrics(_ => infoDetails->getDictFromJsonObject->getArrayFromDict("metrics", []))
+      let metrics =
+        infoDetails
+        ->getDictFromJsonObject
+        ->getArrayFromDict("metrics", [])
+        ->Array.filter(item => {
+          let dict = item->getDictFromJsonObject
+          dict->Dict.get("name")->Belt.Option.getWithDefault(""->Js.Json.string) !==
```

Comment:
Can use `LogicUtils.getString` for extracting a key from dict

Reply:
dict value is json type